pkg/minikube/node/start.go

```diff
@@ -731,3 +737,17 @@ func addCoreDNSEntry(runner command.Runner, name, ip string, cc config.ClusterCo
 
 	return nil
 }
+
+// prints a warning to the console against the use of the 'virtualbox' driver, if alternatives are available
+func warnVirtualBox() {
+	var altDriverList strings.Builder
+	for _, choice := range driver.Choices(true) {
+		if choice.Name != "virtualbox" && choice.Priority != registry.Discouraged && !(choice.Name == "vmware" && !(choice.State.Installed && choice.State.Healthy)) {
```

Comment:
there is no need to single out vmware here.

we should not suggest alternative to Virtualbox in the following cases
- not healthy
- not installed
- discouraged